src/js/components/covid19/amountsVisualization/AmountsVisualization.jsx

```diff
@@ -164,9 +174,16 @@ const AmountsVisualization = ({
                                     scale={scale}
                                     width={width}
                                     tooltipId="_totalOutlays1"
-                                    dataId="_totalOutlays" />
+                                    dataId="_totalOutlays"
+                                    arpStyles={arpStyles} />
                             </svg>
-                            <div className="amounts-viz__sub-title" />
+                            {arpStyles ?
+                                <div className="amounts-viz__sub-title">
+                                    NOTE: Amounts reported in this section were derived using: 1) data tagged as Disaster Emergency Fund Code (DEFC) V spending which was designated for Non-emergency P.L. 117-2, American Rescue Plan; and 2) data manually reported as American Rescue Plan spending for the Department of Labor.
```

Comment:
The [Note shared component](https://github.com/fedspendingtransparency/usaspending-website/blob/qat/src/js/components/sharedComponents/Note.jsx) could be used for this, which should automatically apply the bold styling to "NOTE" that's in the mockup